dns-record-check/src/index.ts

```diff
@@ -0,0 +1,6 @@
+import { expose, util } from '@chainlink/ea-bootstrap'
+import { makeExecute } from './adapter'
+
+const NAME = 'Adex'
+
+export = { NAME, execute: makeExecute, ...expose(util.wrapExecute(makeExecute())) }
```

Comment:
Now adapters only expose `makeExecute`, an `ExecuteFactory` that you can use to build the `Execute` function for the adapter.

```suggestion
export = { NAME, makeExecute, ...expose(util.wrapExecute(makeExecute())) }
```